portfolio/src/main/webapp/script.js

```diff
@@ -45,8 +113,18 @@ function deleteComments(comment) {
   fetch('/delete-comment', {method: 'POST', body: params});
 }
 
-function appendCommentElement(txt, commentDiv) {
+function appendElement(type, txt, commentDiv) {
```

Comment:
'type' is not used - remove?

Reply:
I updated the function so that any type of element can be created with the method like h3's in the showPopup method